src/server/i18n/en.json

```diff
@@ -14,7 +14,8 @@
     "payment": "Payment",
     "active_session": "Active session warning",
     "cookie_preferences": "Cookie preferences",
-    "cookie_details": "Cookie details"
+    "cookie_details": "Cookie details",
+    "accessibility_statement": "Accessibility statement"
```

Comment:
When another language is being used, will it automatically fall back to the English text?

Reply:
This is resolved with https://github.com/dvsa/rsp-public-portal/pull/133/commits/acec263d9ba1a00d14ad45ddf47b284772ef11d5